releases.json

```diff
@@ -79,11 +79,12 @@
     "releases": [
         {
             "framework": "TGI",
-            "device": "inf2",
-            "version": "0.0.24",
+            "device": "gpu",
```

Comment:
why are we changing device type?

Reply:
This file is to specify which image config to trigger for the release pipeline, so each time we release a image we need to update this json. Here we just want to re-trigger the release pipeline